src/app/pages/MarginTradePage/components/AddToMarginDialog/index.tsx

```diff
@@ -43,27 +42,33 @@ export const AddToMarginDialog: React.FC<IAddToMarginDialogProps> = ({
   onCloseModal,
 }) => {
   const canInteract = useCanInteract();
+  const {
+    id,
+    trade,
+    loanToken: { id: loanTokenId },
+    collateralToken: { id: collateralTokenId },
+  } = item;
+  const tradeData = trade as Trade[];
```

Comment:
Are you sure that `trade` is defined? I would probably do it this way - `const entryLeverage = trade?.[0].entryLeverage || 1;` (you can replace `1` with some more suitable fallback value). This is much safer way and without casting. Do the same for `positionSize`.

Reply:
yeah thanks, applied